_data/docsarchive/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM starefossen/github-pages
+# The branch name to check out, also used in some find/replace
+# This is just the default, specify at build-time using --build-arg VER=<foo>
+ARG VER="v1.11"
+
+# Get the branch contents without history
+RUN echo "Started $VER" \
+    && git clone --depth 1 --branch $VER https://github.com/docker/docker.github.io docs_src \
+    && jekyll build -s docs_src -d docs \
+    && rm -rf docs_src \
+    && find docs -type f -name '*.html' -print0 | xargs -0 sed -i 's#href="https://docs.docker.com/#href="/'"$VER"'/#g'
```

Comment:
Don't think they are super problematic, I can see them being useful while we still have some parts of the docs in a separate repo. We need to rewrite here anyway